libs/vkd3d/command.c

```diff
@@ -2645,12 +2621,12 @@ static void d3d12_command_list_clear_attachment_pass(struct d3d12_command_list *
             if (separate_ds_layouts)
             {
                 if (clear_aspects & VK_IMAGE_ASPECT_DEPTH_BIT)
-                    attachment_desc.initialLayout = VK_IMAGE_LAYOUT_UNDEFINED;
+                    initial_layouts[0] = VK_IMAGE_LAYOUT_UNDEFINED;
                 if (clear_aspects & VK_IMAGE_ASPECT_STENCIL_BIT)
-                    stencil_attachment_desc.stencilInitialLayout = VK_IMAGE_LAYOUT_UNDEFINED;
+                    initial_layouts[0] = VK_IMAGE_LAYOUT_UNDEFINED;
```

Comment:
Typo? It seems like this gets addressed in a following commit, but should probably be part of this commit instead.